src/plugins/event-plugins/__tests__/FetchPlugin.test.ts

```diff
@@ -905,4 +907,115 @@ describe('FetchPlugin tests', () => {
             }
         });
     });
+
+    test('when http event is recorded with response then latency is captured', async () => {
+        // Init
+        const resetNow = mockNow();
+        const plugin = new FetchPlugin();
+        plugin.load(xRayOffContext);
+
+        // Run
+        await fetch(URL);
+        plugin.disable();
+
+        // Assert
+        expect(record).toHaveBeenCalledTimes(1);
+        const eventType = record.mock.calls[0][0];
+        const httpEvent = record.mock.calls[0][1] as HttpEvent;
+        expect(eventType).toEqual(HTTP_EVENT_TYPE);
+        expect(httpEvent).toEqual(
+            expect.objectContaining({
+                response: expect.anything(),
+                startTime: expect.any(Number),
+                duration: expect.any(Number)
```

Comment:
issue: Verify that startTime and duration are correct here.

The same applies to the test at line 938, and the tests in *XhrPlugin.test.ts*.

Reply:
Updated to check for specific values. However, this introduces some brittleness because of the way I wrote `mockNow()`. The expected results will be affected by the number of calls to the mocked `Date.now()`.